packages/smart-wallet/src/walletFactory.js

```diff
@@ -171,7 +175,7 @@ export const prepare = async (zcf, privateArgs, baggage) => {
         const wallet = walletsByAddress.get(obj.owner); // or throw
 
         console.log('walletFactory:', { wallet, actionCapData });
-        return E(wallet).handleBridgeAction(actionCapData, canSpend);
+        E(wallet).handleBridgeAction(actionCapData, canSpend);
```

Comment:
Surprised this doesn't need a `void` prefix, `E(wallet)` will always return a promise.

Reply:
Doesn't affect runtime and doesn't affect CI unless `AGORIC_ESLINT_TYPES` is enabled. But it's not https://github.com/Agoric/agoric-sdk/issues/5788

You're right it would be better and I ran locally with that enabled to catch these:
<img width="541" alt="Screenshot 2023-03-06 at 4 42 46 PM" src="https://user-images.githubusercontent.com/21505/223289836-fa9f9564-af04-4101-b841-553e335d1542.png">

Amending and force pushing now.